src/business-logic/services/rundown-timeline-service.ts

```diff
@@ -31,6 +33,13 @@ export class RundownTimelineService implements RundownService {
   ) {}
 
   public async activateRundown(rundownId: string): Promise<void> {
+
+    const basicRundowns: BasicRundown[] = await this.rundownRepository.getBasicRundowns();
+    const activeRundown: BasicRundown | undefined = basicRundowns? basicRundowns.find(rundown => rundown.isActive()) : undefined;
+
+    if (activeRundown) {
+      throw new AlreadyActivatedException(`Unable to activate rundown, because: ${activeRundown.name} is active. `)
+    }
```

Comment:
A nice thing you could do is extract this into a separate method called `assertNoRundownIsActive()`. 
Then you stick to single responsibility :)

Reply:
I think I made what you suggest